### DISCUSSION TITLE
Terminal looks weird and not readable

### DISCUSSION THREAD

**Discussion by tomwach13** [IceWhaleTech/CasaOS#1446] [General]
Hi
When I open terminal in CasaOS it looks weird and is not readable. Not sure if the problem is with CasaOS or some settings in Debian.
Its clean Debian 12 without any desktop environment install with CasaOS. Was trying to google but not sure what to looking for. Any help is appreciable.

![casaos_terminal](https://github.com/IceWhaleTech/CasaOS/assets/48555976/37ebb322-1466-48fd-8ab7-48adbe4961c3)

**Comment by JonasSchubert** [comment]
Hi. That is most likely a browser issue. I have seen similar behaviour in firefox based browsers (Firefox itself,LibreWolf, etc.). But it always worked in Chromium based browsers (e.g. Brave).
Which browser do you use? Maybe try to switch it to one with another engine.

**Comment by Saint6822** [comment]
I know this is an old thread but if someone comes looking the answer to the issue is that in some FF based browsers you need to grant the CasaOS webpage permission to "Extract canvas data".
